bin/check_workspace.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+ROOTDIR=$SCRIPTPATH/..
+cd $ROOTDIR
+
+ret=0
+TMPFILE=$(mktemp)
+
+function cleanup() {
+	rm -f $TMPFILE
+}
+
+trap cleanup EXIT
+
+grep -n commit WORKSPACE  | grep -v "#" > $TMPFILE
+ret=$?
+
+# found a commit line with no comment
+if [[ $ret -eq 0 ]];then
+	cat $TMPFILE
+	echo "Missing comment"
```

Comment:
nit: make error message more useful. maybe something like: "Missing required comment on dependency. Please add comment with date and release info."